oracleservice/oracle.py

```diff
@@ -274,6 +274,8 @@ def _handle_era_change(self, active_era_id: int, era_start_timestamp: int):
                     self._sign_and_send_to_para(tx, stash, active_era_id - 1)
                 else:
                     logger.info(f"Skipping sending the transaction for stash {stash.ss58_address}: oracle is running in debug mode")
+                balance = self.service_params.w3.eth.get_balance(self.account.address)
```

Comment:
If we reload the oracle, the first time we see the oracle's balance is after we send the first transaction. Can we somehow set the oracle balance on the oracle's start?